### PR TITLE
add unshuffle

### PR DESCRIPTION
**Pull request by AmeenUrRehman**
Solving this issue - [ ] #22239 

Hii @YushaArif99 could you review my Pull Request

Hii @YushaArif99  @AnnaTz could u review my pr

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by AmeenUrRehman** [comment]
@YushaArif99 Plz review this

**Comment by AmeenUrRehman** [comment]
@AnnaTz could u plz review this 😢

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_exponents_and_logarithms.py
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_exponents_and_logarithms.py:1]: could you kindly revert this change 🙂

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @AmeenUrRehman 

Apologies for the delay in the review. Was away traveling for a few weeks due to which I couldnt responsive to the PRs 😓 

With regards to your PR, the implementation does look good to me. Could you kindly resolve the merge conflicts and then re-request a review? The CI tests are failing with a weird `dill` import error which I think should go away after the conflicts have been resolved. Thanks! 🙂

**Comment by AmeenUrRehman** [comment]
> Hi @AmeenUrRehman
> 
> Apologies for the delay in the review. Was away traveling for a few weeks due to which I couldnt responsive to the PRs 😓
> 
> With regards to your PR, the implementation does look good to me. Could you kindly resolve the merge conflicts and then re-request a review? The CI tests are failing with a weird `dill` import error which I think should go away after the conflicts have been resolved. Thanks! 🙂

**Comment by AmeenUrRehman** [comment]
Sure, I'll try to resolve this, sorry it closed by mistake.

**Comment by AmeenUrRehman** [comment]
@YushaArif99 hey could u check now.

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @AmeenUrRehman, the conflicts are still showing though 😅. Could you try resolving them once again? Thanks!

**Comment by AmeenUrRehman** [comment]
Sure, working on it.

**Comment by AmeenUrRehman** [comment]
@YushaArif99 , I'll open another pr soon. Their is some problem in my environment

**Pull request closed without merging**